packages/extension-ui/src/components/View.tsx

```diff
@@ -37,7 +37,7 @@ function View({ children, className }: Props): React.ReactElement<Props> {
 
 export const BodyTheme = createGlobalStyle<ThemeProps>`
   body {
-    background-color: ${({ theme }: ThemeProps): string => theme.background};
```

Comment:
[Deleting the border](https://github.com/Cardinal-Cryptography/aleph-zero-signer/pull/80/files#r1204780281) solved the initial problem of the _light_ glow around the edges, but create another problem: a _stronger_ glow the bottom. As expected, the blur filter would _attract_ the white background of the body that was _outside_ of the edges of the application. Now, why the background was white in spite of having a value assigned? Because the assigned value is a gradient and the used property was `background-color` instead of `background`.